src/Module/Install.php

```diff
@@ -218,43 +212,31 @@ protected function content(array $request = []): string
 				break;
 
 			case self::BASE_CONFIG:
-				$ssl_choices = [
-					App\BaseURL::SSL_POLICY_NONE     => $this->t("No SSL policy, links will track page SSL state"),
-					App\BaseURL::SSL_POLICY_FULL     => $this->t("Force all links to use SSL"),
-					App\BaseURL::SSL_POLICY_SELFSIGN => $this->t("Self-signed certificate, use SSL for local links only \x28discouraged\x29")
-				];
+				$baseUrl = $configCache->get('system', 'url') ?
+					new Uri($configCache->get('system', 'url')) :
+					$this->baseUrl;
 
-				$tpl    = Renderer::getMarkupTemplate('install_base.tpl');
+				$tpl    = Renderer::getMarkupTemplate('install/02_base_config.tpl');
 				$output .= Renderer::replaceMacros($tpl, [
 					'$title'      => $install_title,
 					'$pass'       => $this->t('Base settings'),
-					'$ssl_policy' => ['system-ssl_policy',
-						$this->t("SSL link policy"),
-						$configCache->get('system', 'ssl_policy'),
-						$this->t("Determines whether generated links should be forced to use SSL"),
-						$ssl_choices],
-					'$hostname'   => ['config-hostname',
-						$this->t('Host name'),
-						$configCache->get('config', 'hostname'),
-						$this->t('Overwrite this field in case the determinated hostname isn\'t right, otherweise leave it as is.'),
-						$this->t('Required')],
 					'$basepath'   => ['system-basepath',
 						$this->t("Base path to installation"),
 						$configCache->get('system', 'basepath'),
 						$this->t("If the system cannot detect the correct path to your installation, enter the correct path here. This setting should only be set if you are using a restricted system and symbolic links to your webroot."),
 						$this->t('Required')],
-					'$urlpath'    => ['system-urlpath',
-						$this->t('Sub path of the URL'),
-						$configCache->get('system', 'urlpath'),
-						$this->t('Overwrite this field in case the sub path determination isn\'t right, otherwise leave it as is. Leaving this field blank means the installation is at the base URL without sub path.'),
-						''],
+					'$system_url'    => ['system-url',
+						$this->t('The Friendica system URL'),
+						(string)$baseUrl,
+						$this->t('Overwrite this field in case the system URL determination isn\'t right, otherwise leave it as is.'),
```

Comment:
It's ok to use double-quotes to prevent escaping single quotes in text which makes its searching more difficult.
```suggestion
						$this->t("Overwrite this field in case the system URL determination isn't right, otherwise leave it as is."),
```